src/main/java/org/commcare/cases/instance/CaseInstanceTreeElement.java

```diff
@@ -232,38 +196,24 @@ public String getAttributeNamespace(int index) {
 
     @Override
     public String getAttributeName(int index) {
-        if (index == 0) {
-            return "syncToken";
-        } else if (index == 1) {
-            return "stateHash";
-        } else {
-            return null;
-        }
+        return null;
     }
 
     @Override
     public String getAttributeValue(int index) {
-        if (index == 0) {
-            return syncToken;
-        } else if (index == 1) {
-            return stateHash;
-        } else {
-            return null;
-        }
+        return null;
     }
 
     @Override
     public CaseChildElement getAttribute(String namespace, String name) {
-        //Oooooof, this is super janky;
         return null;
     }
 
     @Override
     public String getAttributeValue(String namespace, String name) {
-        return getAttributeValue("syncToken".equals(name) ? 0 : "stateHash".equals(name) ? 1 : -1);
+        return null;
```

Comment:
The default behavior here before was to return -1, so we probably want to preserve that